### PR TITLE
Prune / Organize documentation

### PR DESCRIPTION
**Pull request by DanVanAtta**
- Reduce documentation volume, summarize
- Add team communication page
- Sort the documentation in 'older' and merge it with existing folders, no longer have an 'older' folder. IT contained some high quality docs that were previously bundled with the game install.
- Remove pages that are out of date, or that are just not that badly needed.
- clean up a number of the jekyll page headers

### CODE REVIEW

**Comment by DanVanAtta** [comment]
@RoiEXLab last two commits update the .github contributing links.

File: docs/admin/builder_bot.md [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
 ## Overview
+Bot Account: [github.com/tripleabuilderbot](https://github.com/tripleabuilderbot)
+
 An admin owned account used for automated build tasks that require repository write access.
 
 
 ## Personal Access Keys
-- Travis automated releases key
-  - Usage: [Code Snippet](https://github.com/triplea-game/triplea/blob/master/.travis.yml#L32)
+- *Travis Automated Releases Ley*
```
Comment (ssoloff) [inline docs/admin/builder_bot.md]: "Ley" -> "Key"

File: docs/admin/how_to_release.md [hunk 2]
```diff
@@ -1,96 +1,39 @@
----
-layout: longpage
-title: How to Release
-permalink: /dev_docs/admin/how_to_release
----
 
+## Manual Release Testing
```
Comment (ssoloff) [inline docs/admin/how_to_release.md]: These test descriptions are nice and there are some gems in there (e.g. client should be the first player in a networked game).  Can we document them in `testing.md` and then just link from here?  I wouldn't have thought to look for this level of testing detail under "How to Release."

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,9 +1,12 @@
 <!--
 
-All PRs should follow the standards outlined at: http://www.triplea-game.org/dev_docs/dev/code_standards/
+All PRs should follow the standards outlined at: 
+https://github.com/triplea-game/triplea/blob/master/docs/dev/code_standards.md
 
-More about those standards here: http://www.triplea-game.org/dev_docs/dev/code_format/. 
+More about those standards here: 
+http://www.triplea-game.org/dev_docs/dev/code_format/.
```
Comment (ron-murhammer) [inline .github/PULL_REQUEST_TEMPLATE.md]: Was this doc not moved over?

File: docs/admin/how_to_release.md [hunk 4]
```diff
@@ -1,96 +1,39 @@
...
+  - XML parsing
+  
+- Local host + client
+  - This means start up two copies of triplea.  In one copy you click ‘host networked game’ 
+and start hosting.  In the other copy you click ‘connect to networked game’ and join.  The client should choose the 
+FIRST player in the list (this is to test that the delegates are sending data correctly).  
+Start the game and play a round against yourself, making sure everything looks good.  
+
+- Play by Email, and Play by Forum  
+  - You can test both at once by setting up a game against yourself that has both settings enabled.  You will probably need to add two email addresses for this to work.  I also recommend testing the play by forum against both AxisAndAllies.org and TripleAWarClub.org, and also testing the play by email against both a gmail and a hotmail email account.
```
Comment (ron-murhammer) [inline docs/admin/how_to_release.md]: war club links should pretty much all be removed at this point. PBF should be tested against A&A.org and the new forum.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (DanVanAtta) [inline .github/PULL_REQUEST_TEMPLATE.md]: Incomplete update mistake on my part, it was moved. Now updated: 709cf58

**Comment by DanVanAtta** [comment]
Comments have been addressed. I'll note we are beyond the scope of how things were moved and/or pruned (in the case of the latest updates I was not the original author nor updated in a signfiicant way the text commented, but it is good to get it updated nonetheless).

Since this is documentation only updates, I'm inclined to merge it now to move  forward and end this specific iteration of investment. For the content removed, some of it moved, but the rest I was mostly the original author and am very confident we have that content covered or it is out of date. This gives me confidence to move forward. Any further updates really should be new PRs (the reason behind the change would be different).

**Pull request merged**